src/Murder.Editor/Data/EditorDataManager.cs

```diff
@@ -682,7 +682,23 @@ protected override bool TryCompileShader(string path, [NotNullWhen(true)] out Ef
             }
 
             string binOutputFilePath = FileHelper.GetPath(PackedBinDirectoryPath, string.Format(ShaderRelativePath, path));
-            string arguments = $"/nologo /T fx_2_0 {sourceFile} /Fo {binOutputFilePath}";
+            
+            // NOTE: In Unix systems, absolute paths usually start with a slash:/
+            // which causes the fxc compiler to mistakenly recognize the path parameters as fxc options,
+            // so I replaced the slash at the beginning of the directory with a backslash
+            string paramBinOutputFilePath = binOutputFilePath;
```

Comment:
do we want `string escapedBinOutputFilePath = binOutputFilePath.Replace("/", "\\/")`?

Reply:
```string escapedBinOutputFilePath = binOutputFilePath.Replace("/", "\\/")``` not work.
```string escapedBinOutputFilePath = binOutputFilePath.Replace("/", "\\")``` works.

Reply:
🤔 maybe `FileHelper.EscapePath()`?

Reply:
> 🤔 maybe `FileHelper.EscapePath()`?

It doesn't seem to work

Reply:
maybe I misunderstood the problem then, is it something like:
`/home/path/to/here`
that should be:
`\\home/path/to/here`?

Reply:
> maybe I misunderstood the problem then, is it something like: `/home/path/to/here` that should be: `\\home/path/to/here`?

yes

Reply:
oh okay! I'm surprised that `\\home/path/to/here` works, but that should be ok then

Reply:
> oh okay! I'm surprised that `\\home/path/to/here` works, but that should be ok then
yeah, it looks a bit tricky